crates/telio-dns/src/zone.rs

```diff
@@ -29,11 +29,11 @@ pub type Records = HashMap<String, Vec<IpAddr>>;
 
```

Comment:
As `zone.rs` already have IPv6 support in prod, lets maybe skip the feature flag here.